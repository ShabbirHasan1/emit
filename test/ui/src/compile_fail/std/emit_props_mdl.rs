fn main() {
    emit::emit!("template", mdl: "test");
}
