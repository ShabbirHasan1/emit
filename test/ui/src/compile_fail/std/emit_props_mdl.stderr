error: specify the module using the `mdl` control parameter before the template
 --> src/compile_fail/std/emit_props_mdl.rs:2:29
  |
2 |     emit::emit!("template", mdl: "test");
  |                             ^^^^^^^^^^^
